tests/local_testing/test_assistants.py

```diff
@@ -37,6 +38,63 @@
 - Run Thread -> `/v1/threads/{thread_id}/run`
 """
 
+@pytest.mark.parametrize(
+    "sync_mode",
+    [True, False],
+)
+@pytest.mark.asyncio
+async def test_get_assistants_openai_env(sync_mode, monkeypatch):
+    # See https://github.com/openai/openai-openapi/blob/master/openapi.yaml
+    class MockOpenAIRequestHander(BaseHTTPRequestHandler):
```

Comment:
I used this style to not introduce any new dependencies, but still cover the minimal feature space. If there's a preferred alternative, let me know. I didn't do this for every place the ENV vars are used as it isn't centralized. Basic idea is at least one spot, and the most important is verified that the approach works without a real account.